libs/tools/schematics/interface-asset-schematics/src/create/files/package.json

```diff
@@ -6,10 +6,12 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "npm run cleanDist && npx tsc --project tsconfig.ifd.json && shx cp ./src/package.json ./tmp/ && node --experimental-modules --es-module-specifier-resolution=node tmp/index_ifd.js && npm run cleanTmp && npx webpack --config webpack.config.js",
+    "build:composer": "npm run cleanDist && npx tsc --project tsconfig.ifd.json && shx cp ./src/package.json ./tmp/ && node --experimental-modules --es-module-specifier-resolution=node tmp/index_ifd.js && npm run cleanTmp && npx webpack --config webpack.config.js && npm run postBuild",
```

Comment:
I'm not sure to understand why this command is used for ? Does everybody can use this command ?
Should we add documentation for this ?

Reply:
Same comment. Specific script needed in our own CI for Composer integration should not be in the default template of an IA workspace.
Same for `postBuild` script

Reply:
Indeed, this is only useful for us, when we're building the source code of an IA to integrate it in Composer. Having it here was just making sure that all future IA's will have easy access to these scripts, but it has no purpose for our customers. Any ideas of where I can put them instead?

Reply:
I removed the scripts, we'll just copy what we need manually for further Typescript IAs